protocol/v2/ssv/msgqueue/queue_test.go

```diff
@@ -126,6 +124,35 @@ func generateConsensusMsg(t *testing.T, ssvMsgType spectypes.MsgType, height spe
 	return ssvMsg
 }
 
+func generateDecidedMsg(t *testing.T, ssvMsgType spectypes.MsgType, height specqbft.Height, round specqbft.Round, id spectypes.MessageID, consensusType specqbft.MessageType) *spectypes.SSVMessage {
+	ssvMsg := &spectypes.SSVMessage{
+		MsgType: ssvMsgType,
+		MsgID:   id,
+	}
+
+	commitData, err := (&specqbft.CommitData{
+		Data: []byte("data"),
+	}).Encode()
+
+	require.NoError(t, err)
+
+	signedMsg := specqbft.SignedMessage{
+		Signature: nil,
+		Signers:   []spectypes.OperatorID{1, 2, 3},
+		Message: &specqbft.Message{
+			MsgType:    consensusType,
+			Height:     height,
+			Round:      round,
+			Identifier: nil,
```

Comment:
i think the test will not pass without the identifier  - please check

Reply:
I copied it from `generateConsensusMsg` and the tests seem to pass. I will add an identifier to both `generateConsensusMsg` and `generateDecidedMsg`